assets/js/googlesitekit/modules/index.js

```diff
@@ -30,12 +30,59 @@ import './datastore';
 
 const Modules = {
 	createModuleStore,
+	/**
+	 * Activates Module
+	 *
+	 * Activates a module in the modules store based on the provided args.slug parameter.
+	 *
+	 * @since 1.8.0
+	 *
+	 * @param {Object} [args]      Module arguments.
+	 * @param {string} [args.slug] Slug of the module to activate.
+	 * @return {Object} Object with `{response, error}`. On success, `response.moduleReauthURL`
```

Comment:
See previous comment. This will always return a `Promise` – the object is the value that the promise will be settled with.